policy/score_calculator.go

```diff
@@ -490,22 +468,22 @@ func (c *bandedScoreCalculator) Add(score *Score, impact *explorer.Impact) {
 
 			if category <= 10 {
 				c.critMax += score.Weight
-				if score.Value == 100 {
+				if score.Value < 100 {
```

Comment:
based on my understanding the counter should only increase if the score is not `Pass`

Reply:
based on the `Calculate` function, I think `c.crit` and the rest are supposed to count the number of failures:
```
		if c.crit != 0 {
			res.Value = uint32(math.Floor(float64(50) * pcrCrit))
		} else if c.high != 0 {
			res.Value = uint32(math.Floor(float64(50)*pcrHigh)) + 10
```
If there failures, we need to fall into the first category with failures.

So this all seems right to me